index.js

```diff
@@ -2,7 +2,7 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var coffee = require("coffee-script");
+var coffee = require("coffeescript");
```

Comment:
No need. Peer dep should force it. And it's better to fail hard in this case most likely.